apps/mull-ui/src/app/pages/messages/direct-message/direct-message-chat.tsx

```diff
@@ -0,0 +1,163 @@
+import { IChatForm, ISerializedPost } from '@mull/types';
```

Comment:
One issue i have with this is that its a lot of copy paste from the event-chat page. I've modified this page heavily too in my pr and I wonder how the integration of change will go by